.travis/install_macos.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+brew install python@3 eigen libtar fftw libsquish zlib ffmpeg freetype bullet \
+wxmac opencv assimp libvorbis openssl@1.0 || true
```

Comment:
What is wxmac needed for?  Panda3D does not use wxWidgets.

Reply:
You're right, not sure where I got that idea from.

Reply:
That one's probably on me - I [had CMake checking for WxWidgets](https://github.com/panda3d/panda3d/commit/2d5d1400be58d86a6b42be1dad98b141be74422e) until a week ago.